lxd/device/nic_routed.go

```diff
@@ -68,6 +68,8 @@ func (d *nicRouted) validateConfig(instConf instance.ConfigReader) error {
 		"limits.max",
 		"ipv4.gateway",
 		"ipv6.gateway",
+		"ipv4.routes",
```

Comment:
I think we will also need an additional check further down that ensures that if `ipv{n}.routes` is set that the equivalent `ipv{n}.address` is set also.